Sources/MapboxNavigation/Constants.swift

```diff
@@ -2,12 +2,15 @@ import CoreLocation
 import Foundation
 import MapboxDirections
 
+/**
+ Tuple, which contains collection of coordinates and level of traffic congestion along these coordinates.
+ */
 typealias CongestionSegment = ([CLLocationCoordinate2D], CongestionLevel)
```

Comment:
Maybe at some point we should replace `CongestionSegment` with `Feature` in which the geometry is `LineString` and the `CongestionLevel` is one of the properties. That would be a major change for much later though.